spec/whatif.spec.js

```diff
@@ -4,121 +4,21 @@ let whatIf = require('../src/whatif');
 let WhatIfPromise = require('../src/promise');
 
 describe('WhatIf', () => {
-  it('should be defined', () => {
-    expect(typeof whatIf).toBe('function');
+  let whatIfSpy;
+
+  beforeEach(() => {
+    const wrapper = { WhatIfPromise: WhatIfPromise };
```

Comment:
You could use **object literal property value shorthand** as: `{ WhatIfPromise }`